### PR TITLE
jinja toyaml filter and corresponding refactoring

### PR DESCRIPTION
**Pull request by Kuyantsev-VP**
Added custom filter for jinja for easier creating templates.

### CODE REVIEW

**Comment by rvadim** [comment]
We need to write at least one test for this

**Comment by Kuyantsev-VP** [comment]
> We need to write at least one test for this

Test created. 
After this pull request is merged I will create PR with tests for config readers and config generator.

File: tests/test_jinja_custom_filter.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from config_generator import yaml_filter
+from jinja2 import Environment, FileSystemLoader
+import yaml
+
+
+class TestJinjaCustomFilter:
+    def test_yaml_filter(self):
+        env = Environment(loader=FileSystemLoader('./tests/fixtures/test_jinja'))
```
Comment (rvadim) [inline tests/test_jinja_custom_filter.py]: Use Renderer class for testing

**Review by rvadim — CHANGES_REQUESTED** [review]

File: tests/test_jinja_custom_filter.py (hunk 1 above)
Comment (Kuyantsev-VP) [reply tests/test_jinja_custom_filter.py]: Done

**Review by rvadim — APPROVED** [review]
LGTM

**Review by dkondyrev — APPROVED** [review]

**Pull request merged**